packages/mobile/src/onboarding/registration/EnableBiometry.tsx

```diff
@@ -59,20 +65,28 @@ export default function EnableBiometry({ navigation }: Props) {
         <TopBarTextButton
           title={t('skip')}
           testID="EnableBiometrySkipHeader"
-          onPress={navigateHome}
+          onPress={onPressSkip}
           titleStyle={{ color: colors.goldDark }}
         />
       ),
     })
   }, [navigation, step, totalSteps])
 
+  const onPressSkip = () => {
+    ValoraAnalytics.track(OnboardingEvents.biometric_verification_cancel)
+    navigateHome()
+  }
+
   const onPressUseBiometry = async () => {
     try {
+      ValoraAnalytics.track(OnboardingEvents.biometric_verification_approve)
```

Comment:
For the `approve` and `complete` events (or maybe just the `complete` one) I think it would be nice to have a property that tells us if it was done from this screen or from the Settings screen. It might help us figure out if people are doing it during onboarding or after creating their account more often.

Reply:
i agree! the event names are different though? the one from the onboarding screen is called `biometric_verification_approve` and the one from settings is called `settings_biometric_verification_approve`

Reply:
(From the settings screen, enabling biometrics is via a toggle. Users do not see this EnableBiometry screen again)

Reply:
Oh, my bad, I misread that :)

Reply:
Also, the `sCurrentScreenId` and `sPreviousScreenId` are tracked for all events, so an explicit property is probably not needed.